### PR TITLE
Added isprime function to simple_functions

### PR DESCRIPTION
**Pull request by janvanroestel**
added a fast(ish) isprime function to Roestel/template/simple_functions.py

### CODE REVIEW

**Comment by andersdot** [comment]
I am going to review this

File: Roestel/template/simple_functions.py [hunk 1]
```diff
@@ -10,3 +10,30 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+
+def isprime(n):
+    """Test if an a number is a prime. Method from
```
Comment (andersdot) [inline Roestel/template/simple_functions.py:16]: Very nice header string

File: Roestel/template/simple_functions.py [hunk 2]
```diff
@@ -10,3 +10,30 @@ def factorial(value):
...
+    
+    # check small numbers first
+    if n < 4:
+        return True
+
+    # check if it can be divided by 2 or 3
+    if n%2==0 or n%3==0:
+        return False
+
+    # check rest
```
Comment (andersdot) [inline Roestel/template/simple_functions.py:33]: and beautiful comments along the way, very clear

**Review by andersdot — APPROVED** [review]
I think this looks like a great addition to the function

**Comment by eteq** [comment]
Thanks for opening the pull request and for the review(s)! Everything looks good, so we'll go ahead and merge this. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. 

As just a minor note: Great positivity @andersdot! - it's so easy to get bogged down in the "this needs work" that you forgot to say "great job".

**Pull request merged**